cmd/manager/main.go

```diff
@@ -101,17 +101,23 @@ func main() {
 
 	// Start the Cmd
 	// Update clusteroperator status and log a fatal error
-	fatal(mgr.Start(stopCh), monitorDoneCh, monitorStopCh)
+	exit(mgr.Start(stopCh), monitorDoneCh, monitorStopCh)
 }
 
-// fatal updates clusteroperator status and logs a fatal error
-func fatal(err error, monitorDoneCh <-chan struct{}, monitorStopCh chan struct{}) {
+// exit updates the clusteroperator status and exits with the correct error code.
```

Comment:
Nit: please be consistent about how you write `clusteroperator`. You say `clusterOperator` in some places and `clusterOperator` in others. Pick one and stick to it everywhere.
PS: I am not saying you need to fix all places in this PR but just don't introduce more inconsistencies.

Reply:
Good catch - I had used `cluster operator` for comments in the past and will update these comments to use the same naming convention.

Reply:
Please don't use `cluster operator`. It is one word. I would prefer `ClusterOperator` but am okay with `clusteroperator`.

Reply:
Okay - I'll use `ClusterOperator` and create a PR to update the other files to follow that convention.